dotcom-rendering/src/components/Discussion/Pagination.tsx

```diff
@@ -230,17 +230,17 @@ export const Pagination = ({
 	filters,
 }: Props) => {
 	const totalPages = Math.ceil(topLevelCommentCount / filters.pageSize);
-	// Make decisions aobut which pagination elements to show
+	// Make decisions about which pagination elements to show
 	const showBackButton = totalPages > 4 && currentPage > 1;
-	const showFirstElipsis = totalPages > 4 && currentPage > 3;
+	const showFirstEllipsis = totalPages > 4 && currentPage > 3;
 	const secondPage = decideSecondPage({ currentPage, totalPages });
 	const thirdPage = decideThirdPage({ currentPage, totalPages });
-	const forthPage = decideForthPage({ currentPage, totalPages });
+	const fourthPage = decideFourthPage({ currentPage, totalPages });
```

Comment:
Yeah I think it would be a lot easier to parse if we had a simple array like so:

```ts
const outOfBounds = (page: number) => page < 0 || page > totalPages
const pages = totalPages <= 4 ? [
  1, 2, 3, 4
].filter(outOfBounds)
 : [
  currentPage > 3 ? 1 : undefined,
  currentPage > 3 ? "…" : "…",
  currentPage - 1,
  currentPage,
  currentPage + 1,
  currentPage < totalPages - 3 ? "…" : undefined,
  currentPage < totalPages - 3 ? totalPages : undefined,
].filter(isNonNullable)
```

Not entirely sure what the benefit of such a refactor would be, however… Except maybe for testing?